support-frontend/assets/components/socialShare/socialShare.jsx

```diff
@@ -40,25 +40,25 @@ const socialMedia: {
   [SharePlatform]: SocialMedia,
 } = {
   facebook: {
-    link: 'https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fsupport.theguardian.com%2Fcontribute?acquisitionData=%7B%22source%22%3A%22SOCIAL%22%2C%22campaignCode%22%3A%22component-social-facebook%22%2C%22componentId%22%3A%22component-social-facebook%22%7D&INTCMP=component-social-facebook',
```

Comment:
I tested sharing this old link and it was broken - there was no INTCMP in the URL. The new one works just fine.

Reply:
Great!